### ISSUE TITLE
Can't change title when using PropsPlot class

### ISSUE THREAD

**Issue by ibell** [ibell/coolprop#35]
When using this code, I still end up with the default title.  This is related to issue https://github.com/ibell/coolprop/issues/34

```
from CoolProp.Plots import PropsPlot
import matplotlib.pyplot as plt

PP = PropsPlot('n-Pentane','Ts')
plt.title('Another title')
PP.show()
```

**Comment by ibell** [comment]
This could be resolved by adding a `draw()` method to the PropsPlot class (or better to the base class), which you could use to draw the figure, and then tweak to your heart's content

**Comment by lgpage** [comment]
I was planning on creating simple wrappers for that (title, xlabel, ylabel, etc.) in the base class, so the usage would be:

```
from CoolProp.Plots import PropsPlot

PP = PropsPlot('n-Pentane','Ts')
PP.title('Another title')
PP.show()
```

I'll look at getting this done by the end of the week.
For now you can do the following that should work:

```
from CoolProp.Plots import PropsPlot
import matplotlib

PP = PropsPlot('n-Pentane','Ts')
PP._draw_graph()
ax = PP.axis
ax.set_title('Another title')
matplotlib.pyplot.show()
```

**Comment by ibell** [comment]
+1 for the wrappers!

Yeah I figured out about the _draw_graph() function as well.

**Comment by lgpage** [comment]
Should be fixed with 4a047de0b9b933c7e941ed2b4ee84d62034b8fa4
